beta-src/src/webDiplomacyTheme.ts

```diff
@@ -95,6 +95,30 @@ const webDiplomacyTheme = createTheme({
         },
       },
     },
+    MuiSvgIcon: {
+      styleOverrides: {
+        root: {
+          "&.navIcon": {
+            filter: "drop-shadow(0 0 7px #323232)",
+            margin: 18,
+          },
+          "&.navIconSelected": {
+            filter: "drop-shadow(1px 10px 7px #737373)",
+            height: "52px",
+            width: "53px",
+          },
+        },
+      },
+    },
+    MuiIconButton: {
+      styleOverrides: {
+        root: {
+          "&:hover": {
+            background: "none",
+          },
+        },
```

Comment:
Why do we need this?

Reply:
One of the default style of the MUI Icon Button came with a hover effect that made a ring around the icon it wrapped. ?It looked kinda wonky so I decided to remove the effect until someone decided they wanted to keep it or change it to something else. 

I know my code doesn't show the IconButton wrapper in use now, but I left it in there because I think it's what we'll use to give the NavIcons a button functionality. I can definitely remove it though just for cleanliness sake, it wont affect anything.